.github/workflows/release-embedding-sdk.yml

```diff
@@ -101,7 +101,7 @@ jobs:
 
       - name: Update changelog
         run: |
-          yarn embedding-sdk:generate-changelog
+          yarn embedding-sdk:generate-changelog -o changelog-diff
```

Comment:
I modify the command so that it won't replace the current changelog file. Since we only want the latest change, and we'll use the latest change to prepend the changelog on master.

I've tried the cherry-pick route by using patch, but that [resulted in conflicts](https://github.com/metabase/metabase/actions/runs/9401413295/job/25894060678#step:8:6), so I resorted to prepending changelog diff manually instead.